### PR TITLE
Add AutoClassWeb 1.2.0

### PR DESCRIPTION
**Pull request by pierrepo**
This follows issue #380 

---

 # Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [x] LABEL extra.identifier    
  - [x] LABEL about.documentation   
  - [x] LABEL about.license_file
  - [x] LABEL about.tags   
 
  ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata  

---

Please note that `MAINTAINER` is now deprecated by [Docker](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). To reflect this, I used `LABEL maintainer` instead.

### CODE REVIEW

File: autoclassweb/1.2.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+RUN mkdir -p /app/{config,logs,results}
+
+# Expose volumes and port
+VOLUME /app/config
+VOLUME /app/logs
+VOLUME /app/results
+EXPOSE 5000
+
+# Run the web app
+CMD ["gunicorn", "--config", "gunicorn.py", "flaskapp:app"]
```
Comment (osallou) [inline autoclassweb/1.2.0/Dockerfile]: You should not set entrypoint or cmd as several tools do not manage override for containers and lead to issues to run the co tainer. You should instead document in dockerfile and your readme doc how to launch tool with container

After this, should be ok for merge

**Review by osallou — CHANGES_REQUESTED** [review]

File: autoclassweb/1.2.0/Dockerfile (hunk 1 above)
Comment (pierrepo) [reply autoclassweb/1.2.0/Dockerfile]: This tool is a web application. I do not expect the end-user to run it by hand through the command line, except to start the container itself.

From what I understand, `CMD` could be easily overwritten.

Comment (pierrepo) [reply autoclassweb/1.2.0/Dockerfile]: Tell me if I really need to remove the `CMD` instruction.

Comment (osallou) [reply autoclassweb/1.2.0/Dockerfile]: Plea do remove it to be coherent with other containers
Override is indeed easy, but when integrated with some others tools some do not....
Abd if docume ted, it is easy for user anyway to execute container with  documented cmd line

Comment (pierrepo) [reply autoclassweb/1.2.0/Dockerfile]: OK. Thanks @osallou for your kind advice.
Next commit should be OK :smile: 

Please note that in the mean time, I registered AutoClassWeb to bio.tools.

**Pull request merged**